django/cantusdb_project/main_app/views/chant.py

```diff
@@ -86,9 +86,11 @@ def get_context_data(self, **kwargs):
                     chant.manuscript_full_text, pre_syllabized=False
                 )
                 syls_text, syls_melody = postprocess(syls_text, syls_melody)
-            else:
+            elif chant.incipit:
                 syls_text = syllabize_text(chant.incipit, pre_syllabized=False)
                 syls_text, syls_melody = postprocess(syls_text, syls_melody)
+            else:
+                syls_text = [[""]]
```

Comment:
It doesn't look like you are checking standard spelling in this view (ChantDetail)...is that right?

Reply:
huh - my understanding was that the standard spelling fulltext was not really meant to be displayed with volpiano, but I suppose that is true of the incipit as well. Let me add this - good suggestion!

Reply:
Ok, I think I was responding to:

>  we check for MS spelling full text, then standardized spelling full text, then incipit. Finding none of these, we render the volpiano with no text beneath

and then didn't see it, so sorry if I misunderstood and it's not necessary!

Reply:
I'm a bit confused. I think you identified an actual oversight, and I fixed it in [this commit](https://github.com/DDMAL/CantusDB/pull/576/commits/334573d42cc6d4b7acf2242522a8c0c50cec712f) - is this not the case?

Reply:
No you are correct, you fixed it. I was just explaining why I made the suggestion in the first place (that there seemed to be a discrepancy between your description above and the code), but that if you didn't need to check standardized spelling fulltest, then you didn't need to check standardized spelling fulltext...

so we should consider this, from my perspective, resolved!